css-brochure-form/styles.css

```diff
@@ -0,0 +1,62 @@
+body {
+  width: 400px;
+  margin: 0 auto;
+}
+
+h1 {
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-size: 15px;
+  color: #ec605a;
+  text-transform: uppercase;
+  text-align: center;
+  font-weight: 550;
+  letter-spacing: 0.4px;
+}
+
+input {
+  display: block;
+  width: 400px;
+  font-family: AtlasGrotesk, sans-serif;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  color: #a7a7a7;
+  border-radius: 5px;
+  border-color: #bdc0bf;
```

Comment:
Do you notice how in your example image the border color is different between the sides? Try doing something like this to see if it will fix it.

`border: 1px solid #bdc0bf`